src/components/AddTable.tsx

```diff
@@ -0,0 +1,65 @@
+import Button from "./Button";
+import { ButtonVarient } from "../types/ButtonTypes";
+
+interface Tenant {
+  id: number;
+  name: string;
+  gender: string;
+  dob: string;
+  email: string;
+  mob: string;
+}
+
+const AddTenantTable = () => {
+  const tenantData: Tenant[] = [];
+
+  return (
+    <>
+      <div className="flex justify-between items-center my-7">
+        <h5 className="font-semibold text-xl leading-7">Tenant</h5>
+        <Button
+          label="Add Tenant"
+          image="/assets/plus2.svg"
+          imageAlt="plus-bold-icon"
+          varient={ButtonVarient.DISABLED}
+        />
+      </div>
+      <div className="relative overflow-x-auto shadow-md sm:rounded-lg">
+        <table className="w-full text-left text-sm text-gray-500">
+          <thead className="bg-gray-50 text-xs uppercase text-gray-700 normal-case">
+            <tr>
+              <th className="px-6 py-3">Name</th>
+              <th className="px-6 py-3">Gender</th>
+              <th className="px-6 py-3">DOB</th>
+              <th className="px-6 py-3">Email</th>
+              <th className="px-6 py-3">Mob</th>
+              <th className="px-6 py-3"></th>
+            </tr>
+          </thead>
+          <tbody className="divide-y">
+            {tenantData.length === 0 ? (
+              <tr className="odd:bg-white even:bg-gray-50 bg-white">
```

Comment:
I think that only one background color is enough because we are having only one  table row